ycms/cms/templates/_base.html

```diff
@@ -53,96 +53,89 @@
             {% endfor %}
         </div>
     {% endif %}
-    <div class="lg:flex">
-        <button id="sidebarToggle" class="block lg:hidden text-gray-600 px-4 py-3">
-            <i id="menuIcon" icon-name="menu"></i>
-            <i id="closeIcon" class="hidden" icon-name="x"></i>
-        </button>
-        <div id="sidebar"
-             class="fixed h-full w-64 bg-white border-r shadow-md transform -translate-x-full lg:transform-none lg:block">
-            <a href="{% url 'cms:protected:index' %}" class="block border-b">
-                <img src="{% static 'images/logo-ycms.png' %}"
-                     width="200px"
-                     height="auto"
-                     alt="{% translate " Logo" %}"
-                     class="my-3 mx-auto">
-            </a>
-            <ul>
-                <span class="block mt-5 px-5 text-gray-500 tracking-wide">{% translate "General" %}</span>
-                {% if perms.cms.add_ward %}
-                    <li>
-                        {% url 'cms:protected:zbm' as url %}
-                        <a href="{{ url }}"
-                           class="{% if request.path == url %}selected{% endif %}">
-                            <i icon-name="activity"></i> {% translate "ZBM View" %}
-                        </a>
-                    </li>
-                {% endif %}
+    <div id="sidebar" class="absolute h-full w-64 bg-white border-r shadow-md">
+        <a href="{% url 'cms:protected:index' %}" class="block border-b">
+            <img src="{% static 'images/logo-ycms.png' %}"
+                 width="200px"
+                 height="auto"
+                 alt="{% translate " Logo" %}"
+                 class="my-3 mx-auto">
+        </a>
+        <ul>
+            <span class="block mt-5 px-5 text-gray-500 tracking-wide">{% translate "General" %}</span>
+            {% if perms.cms.add_ward %}
                 <li>
-                    {% url 'cms:protected:patients' as url %}
+                    {% url 'cms:protected:zentral' as url %}
                     <a href="{{ url }}"
                        class="{% if request.path == url %}selected{% endif %}">
-                        <i icon-name="folder-search"></i> {% translate "Manage Patients" %}
+                        <i icon-name="activity"></i> {% translate "ZBM View" %}
```

Comment:
```suggestion
                        <i icon-name="activity"></i> {% translate "Ward List" %}
```
(or `Ward Management`, depending on your earlier choices) - only do this after restoring the `develop` version of this file!